build.gradle

```diff
@@ -299,8 +299,11 @@ configure(opentelemetryProjects) {
 
     jacoco { toolVersion = "0.8.5" }
 
-    googleJavaFormat {
-        toolVersion = '1.7'
+    spotless {
+        java {
+            googleJavaFormat("1.9")
```

Comment:
@bogdandrutu Sorry I forgot people probably aren't familiar with spotless. It's still using google-java-format for formatting Java files. But it can format other types of files, but most importantly, we get automatic license header application with it.